.github/workflows/build.yml

```diff
@@ -6,26 +6,41 @@ name: build
 on: [push, pull_request]
 
 jobs:
-  build:
+  lint:
     runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+      - name: Install pre-commit hook
+        run: |
+          pip install pre-commit
+          pre-commit install
+      - name: Linting
+        run: pre-commit run --all-files
+      - name: Check docstring
+        run: |
+          pip install interrogate
+          interrogate -v --ignore-init-method --ignore-module --ignore-nested-functions --ignore-regex "__repr__" --fail-under 80 mmcls
 
+  build:
+    runs-on: ubuntu-latest
     env:
-      CUDA: 10.1.105-1
-      CUDA_SHORT: 10.1
       UBUNTU_VERSION: ubuntu1804
-      FORCE_CUDA: 1
-      CUDA_ARCH: ${{matrix.cuda_arch}}
     strategy:
       matrix:
         python-version: [3.6, 3.7]
```

Comment:
We can test all pytorch versions with python 3.7, and add extra combinations for python 3.6/3.8 and pytorch 1.6.